src/core/config.go

```diff
@@ -244,6 +244,7 @@ func DefaultConfiguration() *Configuration {
 	config.BuildEnv = map[string]string{}
 	config.Cache.HTTPWriteable = true
 	config.Cache.HTTPTimeout = cli.Duration(25 * time.Second)
+	config.Cache.HTTPConcurrentRequestLimit = 100
```

Comment:
100 seems like probably too many to be useful? I would have thought say 10 concurrent requests is likely enough...